### ISSUE TITLE
Fix this line with `File.separator` in the Save and Load preprocessors

### ISSUE THREAD

**Issue by zaleslaw** [JetBrains/KotlinDL#226]
How about fix this line with `File.separator`? This is not correct for unix systems now.

_Originally posted by @YokiToki in https://github.com/JetBrains/KotlinDL/pull/209#discussion_r711235590_

**Comment by juliabeliaeva** [comment]
There are several places where windows separator are used. Another one is here:
https://github.com/JetBrains/KotlinDL/blob/540bf0316d38d6216ef99b8afd8403cdd72e8889/api/src/main/kotlin/org/jetbrains/kotlinx/dl/dataset/handler/Cifar10Util.kt#L36
I have the patch for these two places, but as it conflicts with changes in #225, I have not send it yet.

There are also usages of windows separators and absolute paths in several jupyter files, those should be fixed as well, maybe in another issue.

**Comment by zaleslaw** [comment]
@juliabeliaeva I merged the #225 .
It will be really helpful to get this patch or PR, but I'm a little bit confused about this separator (I've tested the 0.1 and 0.2 release on Ubuntu and macOS and it seems to work fine on both of them with the same separator).

P.S. I remember this problem with separators a few years ago when I was working on a project which used a file system a lot of the time and it supports multiple filesystems, so I used different separators for each of them.
